### PR TITLE
seiko

### PR DESCRIPTION
**Pull request by seikoodev**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.

### CODE REVIEW

**Comment by Delta-0mega** [comment]
Domain request

File: domains/seiko.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "owner": {
+    "username": "Delta-0mega",
+    "email": "delta-omegaone@proton.me"
+  },
+  "record": {
+    "A": [
+      "38.22.104.142"
```
Comment (DEV-DIBSTER) [inline domains/seiko.json:8]: Nginx Preview page. Can you send a preview of the actual stie?

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

File: domains/seiko.json (hunk 1 above)
Comment (Delta-0mega) [reply domains/seiko.json:8]: I think this is normal. I did the same thing with another domain and it put me on the right page. This is the way micium-hosting uses to connect domain names.

**Comment by Delta-0mega** [comment]
(I added txt record for mail client)

**Review by DEV-DIBSTER — DISMISSED** [review]

**Review by DEV-DIBSTER — APPROVED** [review]

**Pull request merged**